docs/zh_cn/index.rst

```diff
@@ -5,7 +5,7 @@
 
 .. toctree::
    :maxdepth: 2
-   :caption: 快速上手
+   :caption: 预编译包快速上手
```

Comment:
Seems a bit weird. Keeping it as original is Ok to me.

Reply:
done.